GitHubContents.ps1

```diff
@@ -143,3 +153,266 @@ function Get-GitHubContent
 
     return $result
 }
+
+function Set-GitHubContent
+{
+    <#
+    .SYNOPSIS
+        Sets the contents of a file or directory in a repository on GitHub.
+
+    .DESCRIPTION
+        Sets the contents of a file or directory in a repository on GitHub.
+
+        The Git repo for this module can be found here: http://aka.ms/PowerShellForGitHub
+
+    .PARAMETER OwnerName
+        Owner of the repository.
+        If not supplied here, the DefaultOwnerName configuration property value will be used.
+
+    .PARAMETER RepositoryName
+        Name of the repository.
+        If not supplied here, the DefaultRepositoryName configuration property value will be used.
+
+    .PARAMETER Uri
+        Uri for the repository.
+        The OwnerName and RepositoryName will be extracted from here instead of needing to provide
+        them individually.
+
+    .PARAMETER Path
+        The file path for which to set contents.
+
+    .PARAMETER CommitMessage
+        The Git commit message.
+
+    .PARAMETER Content
```

Comment:
It seems that you might want to support two parameter sets...one where the user specifies the `Content` directly, and the other where they specify a `ContentPath` to a file that has the content that they want to use.